lib/chef/provisioning/aws_driver/aws_taggable.rb

```diff
@@ -0,0 +1,18 @@
+module Chef::Provisioning::AWSDriver
+# This module is meant to be included in a resource that is taggable
+# This will add the appropriate attribute that can be converged by the provider
+# TODO it would be nice to not have two seperate modules (taggable/tagger)
```

Comment:
I would like some feedback on this - any ideas for how I could have the resource automatically get a new attribute if I decorate the provider with the `converge_tags` method?